.htmllintrc

```diff
@@ -0,0 +1,12 @@
+{
+  // names of npm modules to load into htmllint
+  "plugins": [],
+
+  "attr-bans": ["align", "background", "bgcolor", "border", "frameborder", "longdesc", "marginwidth", "marginheight", "onclick", "scrolling", "style", "width"],
+  "attr-name-style": "dash",
```

Comment:
Ideally, we could use something like `"attr-name-style": "^(?!on).*"` to disallow *ALL* `on*` attributes, but I'm not clever enough to figure out how `htmllint` wants that defined.

Reply:
Yeah, unfortunately I don't see an `attr-bans-regex` option in the docs, and looking at the [`attr-bans` rule](https://github.com/htmllint/htmllint/blob/21c62bd8321b289b9d92d00265cdb48034d1133c/lib/rules/attr-bans.js), it looks like it's just doing a simple `.forEach()` loop, which doesn't support regexp.

I can try filing an upstream issue to see if they'd consider adding support for regular expression style syntax, or if it's currently possible but not obviously documented.
Currently I think the only way to do it would be to list every "on*" attribute individually.

Reply:
Upstream `attr-bans-regex` request at https://github.com/htmllint/htmllint/issues/236 🐟